### DISCUSSION TITLE
Nested Formula [May 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2118] [Discord Threads]
> 🙋‍♂️ **Barefoot Rob**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Barefoot Rob@discord You can! Do let us know if you find any issues with associated usage.  
#  
> 🙋‍♂️ **Entaum**  
#### This not being available was the deal-breaker for us... the DOCs still say it isn't supported. Is it now supported?  
![image0](https://cdn.discordapp.com/attachments/953117919968112690/974740652376092723/unknown.png)  
#  
> 👨‍🦲 **dstala**  
#### Formula column can refer to another formula column. We will correct documents.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/953117919968112690)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
